automation/Dockerfile

```diff
@@ -3,37 +3,22 @@
 # It enables systemd to be operational.
 FROM centos:7.5.1804
 ENV container docker
-RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
-     systemd-tmpfiles-setup.service ] || rm -f $i; done); \
-    rm -f /lib/systemd/system/multi-user.target.wants/*;\
-    rm -f /etc/systemd/system/*.wants/*;\
-    rm -f /lib/systemd/system/local-fs.target.wants/*; \
-    rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
-    rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
-    rm -f /lib/systemd/system/basic.target.wants/*;\
-    rm -f /lib/systemd/system/anaconda.target.wants/*;
+COPY docker_enable_systemd.sh docker_sys_config.sh ./
+RUN bash ./docker_enable_systemd.sh
 
-# Enable EPEL and centos-release-openstack-rocky repo for new version of pytest.
-RUN yum -y install epel-release centos-release-openstack-rocky
-
-# Enable nmstate test repo for new version of pytest-cov and NetworkManager bug
-# fix of https://bugzilla.redhat.com/show_bug.cgi?id=1642625
-RUN yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/nmstate/nmstate-test-env-el/repo/epel-7/nmstate-nmstate-test-env-el-epel-7.repo && \
-    yum-config-manager --enable nmstate-nmstate-test-env-el
+RUN yum -y install epel-release && \
```

Comment:
These changes are raising several question:
- Why did you placed python2-pip up here and not in the same place that the other python* packages are installed? The location matters in terms of layering images. (I think the 'epel-release' can be included in the CentOS-systemd image, while the rest are specific to the dev env).
- Same remark for the 'pip' part, it probably should be placed in the same `RUN` section as the other python* packages.